src/utils/stackTrace.ts

```diff
@@ -65,14 +63,7 @@ export function captureStackTrace(): ParsedStackTrace {
     const fileName = path.resolve(process.cwd(), frame.file);
     if (isTesting && fileName.includes(path.join('playwright', 'tests', 'config', 'coverage.js')))
       return null;
-    const inClient =
-      // Allow fixtures in the reported stacks.
-      (!fileName.includes('test/index') && !fileName.includes('test\\index')) && (
-        frame.file.includes(path.join('node_modules', 'expect'))
-        || fileName.startsWith(CLIENT_LIB)
-        || fileName.startsWith(CLIENT_SRC)
-        || fileName.startsWith(TEST_LIB)
-        || fileName.startsWith(TEST_SRC));
+    const inClient = fileName.startsWith(CLIENT_LIB) || fileName.startsWith(CLIENT_SRC);
```

Comment:
general observation: I have the feeling that we touch this part too often in the past.